recipes/cyrus-sasl/all/conanfile.py

```diff
@@ -0,0 +1,199 @@
+import glob
+import os
+
+from conans import AutoToolsBuildEnvironment, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class CyrusSaslConan(ConanFile):
+    name = "cyrus-sasl"
+    license = "BSD-4-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.cyrusimap.org/sasl/"
+    description = (
+        "This is the Cyrus SASL API implementation. "
+        "It can be used on the client or server side "
+        "to provide authentication and authorization services."
+    )
+
+    topics = ("SASL", "authentication", "authorization")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "with_openssl": [True, False],
+        "with_cram": [True, False],
+        "with_digest": [True, False],
+        "with_scram": [True, False],
+        "with_otp": [True, False],
+        "with_krb4": [True, False],
+        "with_gssapi": [True, False],
+        "with_plain": [True, False],
+        "with_anon": [True, False],
+        "with_postgresql": [True, False],
+        "with_mysql": [True, False],
+        "with_sqlite3": [True, False],
+    }
+    default_options = {
+        "shared": True,
```

Comment:
```suggestion
        "shared": False,
```
By default, any recipe should use static lib. The only exception occurs if it's not supported.

Reply:
When installing the library with any package manager, static libraries won't be installed. So I thought it may be a good idea to do the same and set the default to shared.